app/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="app_name" translate="false">AntennaPod</string>
+    <string name="log_file_share_exception">No compatible apps found</string>
```

Comment:
Please move this to the `strings.xml` file in the `core` folder, which has all the other translations. That allows our translators to localize the string.